src/js/models/v2/covid19/CoreSpendingTableRow.js

```diff
@@ -9,23 +9,36 @@ const CoreSpendingTableRow = {
     populateCore(data) {
         this._id = data.id || '';
         this.code = data.code || '';
+        this.name = data.name || '';
         this.description = data.description || '';
         this._awardCount = data.award_count || 0;
         this._obligation = data.obligation || 0;
         this._outlay = data.outlay || 0;
         this._faceValueOfLoan = data.face_value_of_loan || 0;
     },
     get obligation() {
+        if (this._obligation === null) {
+            return null;
+        }
         return formatMoney(this._obligation);
     },
     get outlay() {
+        if (this._outlay === null) {
+            return null;
+        }
         return formatMoney(this._outlay);
     },
     get awardCount() {
         return formatNumber(this._awardCount);
     },
     get faceValueOfLoan() {
         return formatMoney(this._faceValueOfLoan);
+    },
+    set outlay(val) {
```

Comment:
we need tests for all these as well; see the PR checklist.

Reply:
Done!